cmd/cashierd/main.go

```diff
@@ -309,18 +314,82 @@ func certStore(config string) (store.CertStorer, error) {
 	return cstore, err
 }
 
+func dropPrivileges(runAsUser string) error {
+	// Lookup privsep user
+	if runAsUser == "" {
+		runAsUser = "nobody"
```

Comment:
Does forcing the program to run as "nobody" make this a breaking change? I don't think it does, but it might be surprising.